Tests/WorkspaceTests/AuthorizationProviderTests.swift

```diff
@@ -126,22 +132,24 @@ final class AuthorizationProviderTests: XCTestCase {
             let configuration = Workspace.Configuration.Authorization(netrc: .user, keychain: .disabled)
             let netrcProvider = try configuration.makeRegistryAuthorizationProvider(fileSystem: fileSystem, observabilityScope: observability.topScope) as? NetrcAuthorizationProvider
 
-            XCTAssertNotNil(netrcProvider)
-            XCTAssertEqual(try netrcProvider.map { try resolveSymlinks($0.path) }, try resolveSymlinks(userPath))
+            let expectedNetrcProvider = try resolveSymlinks(userPath)
+            #expect(netrcProvider != nil)
+            #expect(try netrcProvider.map { try resolveSymlinks($0.path) } == expectedNetrcProvider)
 
-            let auth = netrcProvider?.authentication(for: "https://mymachine.labkey.org")
-            XCTAssertEqual(auth?.user, "user@labkey.org")
-            XCTAssertEqual(auth?.password, "user")
+            let auth = try #require(netrcProvider?.authentication(for: "https://mymachine.labkey.org"))
+            #expect(auth.user == "user@labkey.org")
+            #expect(auth.password == "user")
 
             // delete it
             do {
                 try fileSystem.removeFileTree(userPath)
                 let authorizationProvider = try configuration.makeRegistryAuthorizationProvider(fileSystem: fileSystem, observabilityScope: observability.topScope) as? NetrcAuthorizationProvider
                 // Even if user .netrc file doesn't exist, the provider will be non-nil but contain no data.
-                XCTAssertNotNil(authorizationProvider)
-                XCTAssertEqual(try authorizationProvider.map { try resolveSymlinks($0.path) }, try resolveSymlinks(userPath))
+                let expectedAuthorizationProvider = try resolveSymlinks(userPath)
+                #expect(authorizationProvider != nil)
```

Comment:
Here you'll want to use:

```swift
let authorizationProvider = try #require(authorizationProvider)
```

...which unwraps the optional and gives you back a non-optional, or throws, Then subsequent accesses don't need the force unwrap operator. This is important to avoid crashes in tests.

Reply:
Right. I missed that when I created this PR months ago.  I will update this.